app/models/spree/user_decorator.rb

```diff
@@ -4,7 +4,8 @@
   devise :omniauthable
 
   def apply_omniauth(omniauth)
-    if %w(facebook google_oauth2).include? omniauth['provider']
+    skip_signup_providers = SpreeSocial::OAUTH_PROVIDERS.map { |p| p[1] if p[2] == 'true' }.compact
```

Comment:
Hound and Rubocop have contradictory style rules: one asks for double quotes and the other one asks for single-quoted strings